src/Autocomplete/tests/Functional/AutocompleteFormRenderingTest.php

```diff
@@ -31,7 +31,7 @@ public function testFieldsRenderWithStimulusController()
             ->throwExceptions()
             ->get('/test-form')
             ->assertElementAttributeContains('#product_category', 'data-controller', 'custom-autocomplete symfony--ux-autocomplete--autocomplete')
-            ->assertElementAttributeContains('#product_category', 'data-symfony--ux-autocomplete--autocomplete-url-value', '/test/autocomplete/category_autocomplete_type')
+            ->assertElementAttributeContains('#product_category', 'data-symfony--ux-autocomplete--autocomplete-url-value', '/test/autocomplete/category_autocomplete_type?extra_options=')
```

Comment:
That illustrate my fear, that is something i'd like to avoid... or we just add a massive BC break, changing the URL for all existing autompleters ? 

Or i missread something ?

Reply:
The default value for the `extra_options` is an empty array. In the Form Extension there's
```php
 if ($options['extra_options']) {
```

so it doesn't do anything with already existing URLs if they have no `extra_options` set.

I had to add
```diff
->add('category', CategoryAutocompleteType::class, [
+               'extra_options' => [
+                   'some' => 'cool_option',
+               ],
            ])
```

in the `ProductType` in the `tests/Fixtures` to make Autocomplete add the `extra_options` parameter.

Reply:
Ok I was 100% sure, thank you for thinking of that :)